packages/react/src/components/Popover/index.test.tsx

```diff
@@ -197,13 +172,23 @@ test('onClose should be called, when close button in prompt popover is clicked',
   const handleClose = jest.fn();
   render(<WrapperPrompt tooltipProps={{ onClose: handleClose }} />);
   fireEvent.click(screen.getByText('Close'));
-  await waitFor(() => expect(handleClose).toHaveBeenCalled());
+  await waitFor(() => expect(handleClose).toHaveBeenCalledTimes(1));
+});
+
+test('close opened popover when clicking on the popover trigger button', async () => {
```

Comment:
This test verifies that the content is not in the document once the popover button is clicked. It's failing so we know it's a bug.